### PR TITLE
Homebrew updates for the 1.32 and 1.33 release

### PR DESCRIPTION
**Pull request by tzinsky**
The chapel-release.rb file now matches the currently released homebrew formula and the chapel-main.rb is configured for the 1.33 release.

### CODE REVIEW

**Review by arezaii — APPROVED** [review]

File: util/packaging/homebrew/chapel-release.rb [hunk 1]
```diff
@@ -50,6 +44,8 @@ def install
     (libexec/"chplconfig").write <<~EOS
       CHPL_RE2=bundled
       CHPL_GMP=system
+      CHPL_MEM=cstdlib
+      CHPL_TASKS=fifo
```
Comment (mppf) [inline util/packaging/homebrew/chapel-release.rb:48]: Why are we using cstdlib/fifo here? I don't see a comment explaining it here or in the PR message. I would expect it is a workaround. Is it an old workaround that is no longer needed?

Comment (tzinsky) [reply util/packaging/homebrew/chapel-release.rb:48]: Here was the change that introduced those lines:   https://github.com/chapel-lang/chapel/pull/23415.  We have not tested without these lines but I can remove them and give the formula test.

Comment (tzinsky) [reply util/packaging/homebrew/chapel-release.rb:48]: I will update this to add comments but we need those two lines or the homebrew build fails.   Not sure what the work around is at this point beyond those two lines.

Comment (tzinsky) [reply util/packaging/homebrew/chapel-release.rb:50]: Comments added but will need to raise an issue for fixing this...

**Review by mppf — APPROVED** [review]

**Pull request merged**